sale_stock_cancel_restriction/tests/test_sale_stock_cancel_restriction.py

```diff
@@ -22,8 +21,19 @@ def setUpClass(cls):
         cls.sale_order.action_confirm()
         cls.picking = cls.sale_order.picking_ids
         cls.picking.move_lines.quantity_done = 2
-        cls.picking.button_validate()
```

Comment:
Isn't this changing the test behaviour ?

The picking should be done before entering tests.

Reply:
Previously there was only one setup function in the [test file](https://github.com/OCA/sale-workflow/blob/1c8b405dc7d135c13fa00999ff777937b7390d34/sale_stock_cancel_restriction/tests/test_sale_stock_cancel_restriction.py#L8).

I added two new tests: one to validate and one to cancel. 

It does not change the test behavior.